File: Sum.py
a = int(input("Enter first term (a): "))
d = int(input("Enter common difference (d): "))
n = int(input("Enter number of terms (n): "))

sum_hp = 0

for i in range(n):
    sum_hp += 1 / (a + i * d)

print("Sum of HP series =", sum_hp)

File: Sum.cpp
#include <iostream>
using namespace std;

int main() {
    int a, d, n;
    double sum = 0;

    cout << "Enter first term (a): ";
    cin >> a;

    cout << "Enter common difference (d): ";
    cin >> d;

    cout << "Enter number of terms (n): ";
    cin >> n;

    for(int i = 0; i < n; i++) {
        sum += 1.0 / (a + i * d);
    }

    cout << "Sum of HP series = " << sum;

    return 0;
}
